pkg/auth/rbac.go

```diff
@@ -221,8 +221,7 @@ func CheckUserPermissions(rbacConfig RBACConfig, user *User, env, envGroup, appl
 		}
 	}
 	// The permission is not found. Return an error.
-	p := fmt.Sprintf(PermissionTemplate, user.DexAuthContext.Role, action, envGroup, env, application)
-	return status.Errorf(codes.PermissionDenied, fmt.Sprintf("user does not have permissions for: %s", p))
+	return status.Errorf(codes.PermissionDenied, fmt.Sprintf("The user %s with role %s is not allowed to perform the action %s on environment %s for team %s", user.Name, user.DexAuthContext.Role, action, env, team))
```

Comment:
Please add some quotes:
```suggestion
	return status.Errorf(codes.PermissionDenied, fmt.Sprintf("The user '%s' with role '%s' is not allowed to perform the action '%s' on environment '%s' for team '%s'", user.Name, user.DexAuthContext.Role, action, env, team))
```